advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,590 @@
+
+A new [`--repo`](https://pgbackrest.org/configuration.html#section-repository/option-repo) option has been added to set the repository for a command to operate on. All `repo*-` options are indexed to allow for configuring multiple repositories.
+
+In this section, we'll setup a **demo** cluster (using **EDB Postgres Advanced Server** 13 on CentOS 7) to demonstrate how each pgBackRest command reacts to this new feature.
+
+
+### Configuration
+
+For the purpose of this demonstration, we'll setup two local repositories on our database host. This is **not recommended** for production purpose.
+
+```ini
+[global]
+# repository details
+repo1-path=/var/lib/edb/as13/backups/repo1
+repo1-retention-full=1
+repo2-path=/var/lib/edb/as13/backups/repo2
+repo2-retention-full=1
+
+# general options
+process-max=2
+log-level-console=info
+log-level-file=debug
+start-fast=y
+delta=y
+
+[demo]
+pg1-path=/var/lib/edb/as13/data
+pg1-user=enterprisedb
+pg1-port=5444
+```
+
+For safety reasons, the `--repo` option can't be defined in the configuration file or a warning will be triggered: `WARN: configuration file contains command-line only option 'repo'`. If there is more than one repository configured and the `--repo` option is not specified for a command, the repositories will be acted upon in highest priority order (e.g. `repo1` then `repo2`).
+
+> **_NOTE:_** this new `--repo` option is not required when only `repo1` is configured in order to maintain backward compatibility. However, when a single repository is configured, it is recommended to use `repo1` in the configuration.
+
+
+### Initialization
+
+#### Stanza Create Command
+
+The `stanza-create` command will automatically work with all configured repositories:
+
+```bash
+$ pgbackrest --stanza=demo stanza-create
+P00   INFO: stanza-create for stanza 'demo' on repo1
+P00   INFO: stanza-create for stanza 'demo' on repo2
+```
+
+#### Check Command
+
+The `check` command will trigger a new WAL segment to be archived and try to push it to all defined repositories:
+
+```bash
+$ pgbackrest --stanza=demo check
+P00   INFO: check repo1 configuration (primary)
+P00   INFO: check repo2 configuration (primary)
+P00   INFO: check repo1 archive for WAL (primary)
+P00   INFO: WAL segment ... successfully archived to '...' on repo1
+P00   INFO: check repo2 archive for WAL (primary)
+P00   INFO: WAL segment ... successfully archived to '...' on repo2
+```
+
+
+### Backups
+
+#### Archive Push Command
+
+The `archive-push` command will always archive WALs in all configured repositories but backups will need to be scheduled individually for each repository. In many cases this is desirable since backup types and retention will vary by repository.
```

Comment:
In many cases, this is desirable since backup types and retention will vary by the repository.

Reply:
I think "will vary by repository" is more accurate here. Any English native speaker could check this please (@drsm79)?

Reply:
FWIW, I think 'could vary per repository' is better fitting.

Reply:
Fixed in 7fa8ff1